test/e2e/branding_test.go

```diff
@@ -0,0 +1,164 @@
+package e2e
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	operatorsv1 "github.com/openshift/api/operator/v1"
+	consoleapi "github.com/openshift/console-operator/pkg/api"
+	"github.com/openshift/console-operator/pkg/testframework"
+)
+
+// Test the customization-branding features of the config map
+func setupBrandingTestCase(t *testing.T) *testframework.Clientset {
+	client := testframework.MustNewClientset(t, nil)
+	return client
+}
+
+// the simpler test first would be branding set via the operator config
+func TestOperatorConfigBranding(t *testing.T) {
+	validBrands := []operatorsv1.Brand{operatorsv1.BrandOKD, operatorsv1.BrandOCP, operatorsv1.BrandOnline, operatorsv1.BrandDedicated}
```

Comment:
since this is common for both TCs lets pull it out of them
```go
var validBrands = []operatorsv1.Brand{operatorsv1.BrandOKD, operatorsv1.BrandOCP, operatorsv1.BrandOnline, operatorsv1.BrandDedicated}
```